### PR TITLE
Fixes #5052 - Messaging on settings page

### LINKED ISSUE #5052: [FxA] Messaging on settings page
Opening issue to discuss messaging on settings page to educate user what disconnect / sign out does

settings-logout

### PR DESCRIPTION
**Pull request by punamdahiya**

### CODE REVIEW

**Comment by 6a68** [comment]
This can't land until we have product approval on the strings

File: locales/en-US/server.ftl [hunk 1]
```diff
@@ -280,6 +280,7 @@ settingsDescription = You can sign in with Firefox Accounts to sync all your scr
 settingsPageSubHeader = Sync & Accounts
 settingsClosePreferences =
     .title = Close preferences
+settingsFxaDisconnectDescription = If you disconnect with Firefox Accounts your synced screenshots will remain in your account but you will need to signin to regain access to them.
```
Comment (johngruen) [inline locales/en-US/server.ftl]: If you sign out, you will need to sign in again to regain access to your screenshots.

File: locales/en-US/server.ftl [hunk 2]
```diff
@@ -280,6 +280,7 @@ settingsDescription = You can sign in with Firefox Accounts to sync all your scr
 settingsPageSubHeader = Sync & Accounts
```
Comment (johngruen) [inline locales/en-US/server.ftl]: This should not be Sync and Accounts...
It should just be `Firefox Account`

Comment (johngruen) [reply locales/en-US/server.ftl]: We should not use the word Sync here, Sync is a specific service name and we should not conflate the two

**Comment by punamdahiya** [comment]
Updated with product feedback
screenshot_2018-11-15 settings

**Comment by punamdahiya** [comment]
@flodolo Including to review localization changes. Thanks!

**Comment by 6a68** [comment]
LGTM, needs a rebase though

**Comment by punamdahiya** [comment]
Landed after rebasing with latest master

**Pull request merged**